realtime_tools/CMakeLists.txt

```diff
@@ -80,29 +80,27 @@ if(BUILD_TESTING)
   target_link_libraries(realtime_buffer_tests realtime_tools)
 
   ament_add_gmock(lock_free_queue_tests test/lock_free_queue_tests.cpp)
-  if(WIN32)
-    # atomic is not found on Windows, but also not needed
-    target_link_libraries(lock_free_queue_tests realtime_tools Boost::boost)
-  else()
-    # without adding atomic, clang throws a linker error
-    target_link_libraries(lock_free_queue_tests realtime_tools atomic Boost::boost)
+  target_link_libraries(lock_free_queue_tests realtime_tools ${Boost_LIBRARIES})
```

Comment:
Is there a reason going for `${Boost_LIBRARIES}` instead of `Boost::boost`? The latter seems to be what modern CMake targets are supposed to look like.

Reply:
I used `Boost::boost` at first and changed it later. I can modify it